cider/tests/hashtable/CiderHashTableTest.cpp

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include <gtest/gtest.h>
+#include <algorithm>
+#include <iostream>
+#include <random>
+#include <unordered_map>
+#include <vector>
+#include "exec/operator/join/CiderLinearProbingHashtable.h"
+#include "exec/operator/join/CiderStdUnorderedHashTable.h"
+#include "util/Logger.h"
+
+// hash function for test collision
+struct Hash {
```

Comment:
`Hash` and `murmurHash` could be default template parameter for `LPHashTable`? 
Please tell your user how to construct them.

Reply:
I am planning make all hash functions in a seperate folder. This murmurHash is just for test or first version integration